commons-numbers-complex-arrays/src/main/java/org/apache/commons/numbers/complex/arrays/ComplexList.java

```diff
@@ -28,23 +28,9 @@
 import java.util.Objects;
 
 /**
- * Resizable-double array implementation of the List interface. Implements all optional list operations,
```

Comment:
Some of this javadoc still applies. The list provides efficient storage of complex numbers. It does not accept null complex objects.